esphomeyaml/cookbook/bme280_environment.rst

```diff
@@ -0,0 +1,117 @@
+BME280 Environment
+==================
+
+.. seo::
+    :description: Instructions for setting up BME280 sensors in esphomelib and calculate altitude, absolute humidity, and sea level pressure.
+    :image: bme280.jpg
+    :keywords: BME280
+
+The :doc:`</esphomeyaml/components/sensor/bme280>` is a simple temperature, humidity, and pressure sensor with communication over I2C.
+With some simple math it is possible to either determine the height of the sensor, or the current pressure at sea level.
+This guide can be applied to any sensor measuring temperature and pressure at the same time, like the :doc:`</esphomeyaml/components/sensor/bmp280>`, or :doc:`</esphomeyaml/components/sensor/bme680>`.
+
+.. figure:: images/bme280-header.jpg
+    :align: center
+    :width: 75.0%
+
+The first step is to connect the sensor as described :doc:`here </esphomeyaml/components/switch/bme280>`.
+After validating the sensor is working, we can proceed and add some formulas.
+
+.. code-block:: yaml
+    globals:
+      - id: standard_sea_level_pressure_hpa
+        type: float
+        restore_state: no
+        initial_value: '1013.25'
+
+    sensor:
+      - platform: bme280
+        temperature:
+          name: "BME280 Temperature"
+          id: bme280_temperature
+        pressure:
+          name: "BME280 Pressure"
+          id: bme280_pressure
+        humidity:
+          name: "BME280 Humidity"
+          id: bme280_humidity
+        address: 0x77
+        update_interval: 15s
+      - platform: template
+        name: "Altitude"
+        lambda: >-
```

Comment:
use `lambda: |-` (it's safer for C++; I know the docs recommended `>-`, but that has been fixed in the edge version)